ArcGISRuntimeSDKQt_CppSamples/Geometry/ListTransformations/README.md

```diff
@@ -34,18 +34,16 @@ The map starts out zoomed into the grounds of the Royal Observatory, Greenwich.
 
 ## Additional information
 
-This sample uses a `GeographicTransformation`, which extends the `DatumTransformation` class. As of 100.9, ArcGIS Runtime also includes a `HorizontalVerticalTransformation`, which also extends `DatumTransformation`. The `HorizontalVerticalTransformation` class is used to transform coordinates of z-aware geometries between spatial references that have different geographic and/or vertical coordinate systems.
+This sample uses a `GeographicTransformation`, which extends the `DatumTransformation` class. As of 100.9, the ArcGIS Maps SDK for Qt also includes a `HorizontalVerticalTransformation`, which also extends `DatumTransformation`. The `HorizontalVerticalTransformation` class is used to transform coordinates of z-aware geometries between spatial references that have different geographic and/or vertical coordinate systems.
 
 Some transformations aren't available until transformation data is provided.
 
 This sample can be used with or without provisioning projection engine data to your device. If you do not provision data, a limited number of transformations will be available.
 
-This sample uses a `GeographicTransformation`, which extends the `DatumTransformation` class. As of 100.9, ArcGIS Runtime also includes a `HorizontalVerticalTransformation`, which also extends `DatumTransformation`. The `HorizontalVerticalTransformation` class is used to transform coordinates of z-aware geometries between spatial references that have different geographic and/or vertical coordinate systems.
-
 To download projection engine data to your device:
 1. Log in to the ArcGIS for Developers site using your Developer account.
 2. In the Dashboard page, click 'Download APIs and SDKs'.
-3. Click the download button next to `ArcGIS Runtime Coordinate System Data 100.x` to download projection engine data to your computer.
+3. Click the download button next to `ArcGIS Maps SDK for Native Apps Coordinate System Data 100.x` to download projection engine data to your computer.
```

Comment:
this would be worth bringing up in the product renaming slack channel. Not sure if we've considered how this deliverable will be renamed